morebits.css

```
@@ -114,18 +114,18 @@ form.quickform .quickformSubgroup
 	margin-left: 3em;
 }
 
-form.quickform .tooltipButtonContainer
+/* The tooltip button and the content itself */
+form.quickform .morebits-tooltipButton
 {
-	position: relative;
-	width: 100%;
-}
-
-form.quickform .tooltipButton
-{
-	color: blue;
+	color: mediumblue;
 	font-weight: bold;
 	cursor: help;
-	padding: .2em;
+	padding: .3em;
+}
+.morebits-ui-tooltip
+{
+	padding: 4px 6px 4px 6px;
+	font-size: 1.2em;
```

The font size within the tooltip is ENORMOUS at my end - looks like about 18px, not easily measurable.

---

19.2, yeah.  This is annoying skin shit.  I was aiming for 12 px, but they behave differently.  For a quick example:

- Vector: `0.8em` of `100%` = 12.8px
- Timeless: `1.1em` of 0.95em` = 16.72px
- Modern: `1.1em` of `x-small` = 11px

There's a second issue, though.  12px looks a little small on Vector, because it has `font-family: sans-serif` whereas 13px is a little big on Modern and Timeless, because they both have `font-family: Verdana,Arial,sans-serif`.

YMMV, but IMO `13px` looks better on Modern and Timeless than `12px` does on Vector (and we should be favoring Vector anyway).  `rem` and `%` are no help AFAICT.

```suggestion
	font-size: 13px;
```

---

The default (without any `font-size` set) looks good to me on vector, monobook and modern. For timeless, it's huge but easily restored back to normal by putting something like
```css
.skin-timeless .morebits-ui-tooltip {
	font-size: 13px;
}
```